pkg/sql/plan/function/function_id.go

```diff
@@ -402,6 +403,7 @@ var functionIdRegister = map[string]int32{
 	"extract":                        EXTRACT,
 	"if":                             IFF,
 	"iff":                            IFF,
+	"curdate":                        CUR_DATE,
```

Comment:
if it's the same as current_date.  maybe you should use 'curdate': CUREENT_DATE.
and register in builtins.go in one time for two same functions.

Reply:
OK